docs/README.md

```diff
@@ -0,0 +1,203 @@
+
+| TravisCI Environment Variable       | Desired Capabilities Variable  | Description                                         | Default Value                                             |
+|-------------------------------------|--------------------------------|-----------------------------------------------------|-----------------------------------------------------------|
+| KOBITON_DEVICE_PLATFORM_NAME        | platformName                   | Kobiton Device Platform Name (e.g: Android, iOS)    | Android                                                   |
+| KOBITON_DEVICE_NAME                 | deviceName                     | Kobiton Device Name                                 | Galaxy*                                                   |
+| KOBITON_DEVICE_PLATFORM_VERSION     | platformVersion                | Kobiton Device Platform Version                     | *No*                                                      |
+| KOBITON_SESSION_DEVICE_ORIENTATION  | deviceOrientation              | Device Orientation (e.g: Portrait, Landscape)       | portrait                                                  |
+| KOBITON_SESSION_CAPTURE_SCREENSHOTS | captureScreenshots             | Enable screenshots capture during session execution | true                                                      |
+| KOBITON_SESSION_DEVICE_GROUP        | deviceGroup                    | Kobiton device group                                | KOBITON                                                   |
+| KOBITON_ORGANIZATION_GROUP_ID       | groupId                        | Group ID of the device                              | *No*                                                      |
+| KOBITON_SESSION_APPLICATION_URL     | app                            | Download link to the application used for testing   | https://appium.github.io/appium/assets/ApiDemos-debug.apk |
+
+For example, if desired capabilities for executing automation test of the provided demo Android application on Pixel 2 XL running Android 8.1.0:
+
+```javascript
+var desiredCaps = { 
+  sessionName:        'Sample Automation Test Session',
+  sessionDescription: 'This is a sample automation test session', 
+  deviceOrientation:  'portrait',
+  captureScreenshots: false, 
+  app:                'https://appium.github.io/appium/assets/ApiDemos-debug.apk',
+  deviceGroup:        'KOBITON', 
+  groupId:            100,
+  deviceName:         'Pixel 2 XL',
+  platformVersion:    '8.1.0',
+  platformName:       'Android' 
+}
+```
+
+The environment variables representing above desired capabilities should look like this:
+
+![Desired Caps To ENV Example](./assets/desired-caps-travisci-env.png)
+
+> More information about Desired Capabilities and its parameters can be found in https://docs.kobiton.com/automation-testing/desired-capabilities-usage/
+
+#### 2.4. TravisCI configuration file
```

Comment:
```
#### 2.4. Explain TravisCI configuration file

Travis CI can run on this repo since we already provide a simple config file at [.travis.yml](..). Below is the brief on what the config file, you can skip this section if you're familiar in Travis CI

...

```